app/assets/javascripts/utility.js

```diff
@@ -55,14 +55,18 @@
   };
 
   microp.format.removeGitPrefix = function(str) {
-    return str.split('https://github.com/').join('');
+    return str.includes('https')
```

Comment:
this is all fine, but maybe we should eventually extract a standalone js library for this stuff.